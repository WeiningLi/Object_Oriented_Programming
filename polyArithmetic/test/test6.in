create a 1/2 5 2/1 2 1/2 1 1/2 0
print a 
create b 2/1 5 1/2 4 2/1 2 1/2 1 1/4 0
print b
create c 0/1 0
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
multiply a b
quit
