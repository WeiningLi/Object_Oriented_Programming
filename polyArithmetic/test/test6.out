(1/2)x^5 + (2)x^2 + (1/2)x + (1/2)
(2)x^5 + (1/2)x^4 + (2)x^2 + (1/2)x + (1/4)
0
(-3/2)x^5 + (-1/2)x^4 + (1/4)
(5/2)x^5 + (1/2)x^4 + (4)x^2 + (1)x + (3/4)
(1/2)x^5 + (2)x^2 + (1/2)x + (1/2)
(1/2)x^5 + (2)x^2 + (1/2)x + (1/2)
(1/2)x^5 + (2)x^2 + (1/2)x + (1/2)
(-1/2)x^5 + (-2)x^2 + (-1/2)x + (-1/2)
(1)x^10 + (1/4)x^9 + (5)x^7 + (9/4)x^6 + (11/8)x^5 + (17/4)x^4 + (2)x^3 + (7/4)x^2 + (3/8)x + (1/8)
